redis_broadcast.go

```diff
@@ -118,12 +118,10 @@ func newRedisBroadcast(nsp string, adapter *RedisAdapterOptions) (*redisBroadcas
 	if err != nil {
 		return nil, err
 	}
-
 	bc.pub = redis.PubSubConn{Conn: pub}
 	bc.sub = redis.PubSubConn{Conn: sub}
-
 	bc.nsp = nsp
-	bc.uid = uuid.NewV4().String()
+	bc.uid = uuid.Must(uuid.NewV4()).String()
```

Comment:
`uuid.Must(uuid.NewV4()).String()`

i am suggest this logic to function and call where you need